run-tests.php

```diff
@@ -74,27 +74,6 @@ function main()
 	}
 	define('TEST_PHP_SRCDIR', getcwd());
 
-
-	/* Sanity check to ensure that pcre extension needed by this script is available.
-	 * In the event it is not, print a nice error message indicating that this script will
-	 * not run without it.
-	 */
-
-	if (!extension_loaded('pcre')) {
```

Comment:
I tried to remove this once, but it didn't get approved #3105 😕

Reply:
I'm not sure how it could be disabled in PHP-7.3 either... @krakjoe just a friendly ping, that now, we'll be removing this in 7.4. PCRE cannot be disabled via configuration options or without physical removal of the folder and manual patches of php...